api/environments/identities/models.py

```diff
@@ -141,6 +141,32 @@ def get_all_feature_states(
 
         return list(identity_flags.values())
 
+    def get_overridden_feature_states(self) -> dict[str, FeatureState]:
+        """
+        Get all overridden feature states for an identity.
+
+        :return: dict[str, FeatureState] - Key: feature ID. Value: Overridden feature_state.
+        """
+        overridden_feature_states: dict[str, FeatureState] = {}
```

Comment:
This part needs updating too.
```suggestion
        overridden_feature_states: dict[int, FeatureState] = {}
```

Reply:
Correct, my changes will include also that, thanks!

Reply:
Usually the suggestions breke code since are amde on the fly without an IDE so I apply them in my IDE checking what else is needed, and usually additional changes are needed :)